shielder/contract/drink_tests/utils/psp22.rs

```diff
@@ -0,0 +1,74 @@
+use drink::{
+    runtime::MinimalRuntime,
+    session::{Session, NO_ENDOWMENT, NO_SALT},
+    AccountId32, ContractBundle,
+};
+
+pub fn deploy_test_token(
+    session: &mut Session<MinimalRuntime>,
+    supply: u128,
+) -> Result<AccountId32, Box<dyn std::error::Error>> {
+    let formatted_supply: String = format!("{}", supply);
+
+    let psp22_bundle =
+        ContractBundle::load(std::path::Path::new("../PSP22/target/ink/psp22.contract"))?;
```

Comment:
this requires local building, right? if so, then please provide... maybe a Makefile(?) that prepares all the test resources and runs them; it could be then used in CI

Reply:
added makefile